htdocs/avoloicustomtypes/class/api_avoloicustomtypes.class.php

```diff
@@ -233,4 +233,197 @@ public function tierstype($id) {
 			throw new RestException(405, "Problème lors de la récupération du type de tiers");
 		}
 	}
+
+	/////////////// PARTIE TYPE DE RENDEZ-VOUS ///////////////
+
+	/**
+	 * Create an event type
+	 * 
+	 * @param 	string    $eventType		Event type
+	 * @param		string		$color_code		Color of the event type
+	 * @return  string									Created type ID
+	 *
+	 * @throws 500
+	 * @throws 501
+	 * @throws 400
+	 * @throws 401
+	 * @throws 404
+	 * @throws 200
+	 *
+	 * @url POST /createeventtype
+	 */
+	public function createeventtype($eventType, $color_code = "FFC266") {
+		global $conf, $langs, $user;
+
+		$libelle = $eventType;
+
+		// Création du code
+		$code = $this->generaterandomcode();		
```

Comment:
CF mon commentaire sur le camelCase

```suggestion
		$code = $this->generateRandomCode();		
```